.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: 'CI'
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - 'releases/*'
+
+jobs:
+  check-lf-files:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Java 17
+        run: |
+          echo "$JAVA_HOME_17_X64/bin" >> $GITHUB_PATH
+          echo "org.gradle.java.home=${JAVA_HOME_17_X64//\\/\/}" >> gradle.properties
```

Comment:
I copied this from elsewhere, but in hindsight, this doesn't seem right to me. At the cwd, there is no `gradle.properties`...

Reply:
Looks like we don't need it...